src/metabase/documents/schema.clj

```diff
@@ -0,0 +1,9 @@
+(ns metabase.documents.schema
+  (:require
+   [metabase.lib.schema.id :as lib.schema.id]
+   [metabase.util.malli.registry :as mr]))
+
+(mr/def ::document
+  "Schema for an instance of a `:model/Document`."
+  [:map
+   [:id ::lib.schema.id/document]])
```

Comment:
(I'm also questioning the wisdom of having Document ID be something that lives inside of Lib... Documents really aren't a Lib concern, are they? I would put the `::id` schema for a Document **here**... but I guess that ID schema wasn't added to Lib in this PR. Feel free to move it tho and save me some cleanup effort later)